src/NServiceBus.Core.Tests/Scheduler/ScheduleTests.cs

```diff
@@ -36,12 +36,23 @@ public void When_scheduling_an_action_with_a_name_the_task_should_get_that_name(
         }
 
         [Test]
+        public void When_scheduling_an_anonymous_action_without_a_name_the_task_should_get_the_DeclaringType_as_name()
```

Comment:
this is due to the roslyn related changes of the declaringType. I took the fix and unit tests from v6.